specs/airr-schema.yaml

```diff
@@ -2046,10 +2104,14 @@ Diagnosis:
                         id: DOID:4
                         label: disease
         disease_length:
-            type: string
+            $ref: '#/PhysicalQuantity'
```

Comment:
See above -> `TimeInterval`

Reply:
Created TimeQuantity and made the change